stress/common/src/main/java/alluxio/stress/PinListFileSystemMasterClient.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress;
+
+import alluxio.conf.PropertyKey;
+import alluxio.grpc.FileSystemMasterWorkerServiceGrpc;
+import alluxio.grpc.GetPinnedFileIdsPRequest;
+import alluxio.master.MasterClientContext;
+import alluxio.worker.file.FileSystemMasterClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Helper client that is used to minimizes the deserialization overhead by calling
+ * getPinnedFileIdsCount instead of getPinnedFileIdsList
+ */
+public class PinListFileSystemMasterClient extends FileSystemMasterClient {
```

Comment:
I moved the class into `GetPinnedFileIdsBench` as an inner class, since it's unlikely there would be anywhere else where a `.getPinListLength()` call would more useful than `.getPinList().size()`.